config/folio/locations.csv

```diff
@@ -1,571 +1,571 @@
-id,name,code,institutionCode,campusCode,libraryCode,discoveryDisplayName,isActive,primaryServicePointCode,servicePointCodes,description,details.pageAeonSite,details.pageMediationGroupKey,details.pageServicePointCodes,details.pageServicePoints,details.availabilityClass,details.holdingsTypeName,details.scanServicePointCode,details.scanMaterialTypeNames,details.scanPseudopatronBarcode,details.pageSchedule,details.pageServicePointIds,details.pagingSchedule,details.searchworksTreatTemporaryLocationAsPermanentLocation
```

Comment:
It looks like we've lost some columns here.

Reply:
That's expected as Sarah fixes up our mistakes.